packages/create-sitecore-jss/src/templates/nextjs-styleguide/data/routes/styleguide/{{language}}.yml

```diff
@@ -7,20 +7,20 @@ placeholders:
       heading: JSS Styleguide
       content: |
         <div class="alert alert-primary">
-          This is a partial Danish translation of the JSS Styleguide to demonstrate translation functionality.
+          This is a partial <%- language %> translation of the JSS Styleguide to demonstrate translation functionality.
           Use the <a href="/en/styleguide">English version</a> for full content.
         </div>
-        <p>Indholdet og layoutet på denne side er defineret i <code>/data/routes/styleguide/da-DK.yml</code></p>
+        <p>The content and layout of this page is defined in <code>/data/routes/styleguide/<%- language %>.yml</code></p>
   - componentName: Styleguide-Layout
     placeholders:
      <%- appPrefix ? `${helper.getPascalCaseName(appName)}-` : "" %>jss-styleguide-layout:
       - componentName: Styleguide-Section
         fields:
-          heading: Flersprogede mønstre
+          heading: Multilingual Patterns
         placeholders:
          <%- appPrefix ? `${helper.getPascalCaseName(appName)}-` : "" %>jss-styleguide-section:
           - componentName: Styleguide-Multilingual
             fields:
-              heading: Brug af ordbogen
-              sample: Denne tekst kan oversættes til da-DK.yml
+              heading: Translation Patterns
+              sample: This text can be translated in <%- language %>.yml
 
```

Comment:
Should remove the " - or not using Danish" portion of the comment here: https://github.com/Sitecore/jss/blob/902397130eae7a9cc733ab87db2f227b21421762/packages/create-sitecore-jss/src/templates/nextjs-styleguide/data/routes/%7B%7Blanguage%7D%7D.yml#L2

Reply:
Good catch!